rocAL/rocAL_pybind/example/new_api/TEST_FILE.sh

```diff
@@ -0,0 +1,260 @@
+
+CURRENTDATE=`date +"%Y-%m-%d-%T"`
+
+# Mention the number of gpus
+gpus_per_node=1
+
+# Mention Batch Size
+batch_size=10
+
+# python version
+ver=$(python -V 2>&1 | sed 's/.* \([0-9]\).\([0-9]\).*/\1\.\2/')
+
+
+####################################################################################################################################
+# USER TO MAKE CHANGES HERE FOR TEST
+# Make the respective " Pipeline " to test equal to 1
+rocAL_api_python_unittest=1
+rocAL_api_coco_pipeline=0
+rocAL_api_caffe_reader=0
+rocAL_api_caffe2_reader=0
+rocAL_api_tf_classification_reader=0
+rocAL_api_tf_detection_pipeline=0
+rocAL_api_video_pipeline=0
+####################################################################################################################################
+
+
+
+
+####################################################################################################################################
+if [[ rocAL_api_python_unittest -eq 1 ]]; then
+
+    # Mention dataset_path
+    data_dir=$ROCAL_DATA_PATH/images_jpg/labels_folder/
+
+
+    # rocAL_api_python_unittest.py
+    # By default : cpu backend, NCHW format , fp32
+    # Please pass image_folder augmentation_nanme in addition to other common args
+    # Refer rocAL_api_python_unitest.py for all augmentation names
+    python$ver rocAL_api_python_unittest.py \
```

Comment:
please add "" for the $ver. It is generally better to define variables with double quotes

Reply:
Resolved